Microsoft.Azure.CosmosRepository/src/Providers/DefaultCosmosItemConfigurationProvider.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.CosmosRepository.Options;
+
+namespace Microsoft.Azure.CosmosRepository.Providers
+{
+    class DefaultCosmosItemConfigurationProvider : ICosmosItemConfigurationProvider
+    {
+        private static readonly ConcurrentDictionary<Type, ItemOptions> ItemOptionsMap = new();
+
+        private readonly ICosmosContainerNameProvider _containerNameProvider;
+        private readonly ICosmosPartitionKeyPathProvider _cosmosPartitionKeyPathProvider;
+        private readonly ICosmosUniqueKeyPolicyProvider _cosmosUniqueKeyPolicyProvider;
+
+        public DefaultCosmosItemConfigurationProvider(
+            ICosmosContainerNameProvider containerNameProvider,
+            ICosmosPartitionKeyPathProvider cosmosPartitionKeyPathProvider,
+            ICosmosUniqueKeyPolicyProvider cosmosUniqueKeyPolicyProvider
+        )
+        {
+            _containerNameProvider = containerNameProvider;
+            _cosmosPartitionKeyPathProvider = cosmosPartitionKeyPathProvider;
+            _cosmosUniqueKeyPolicyProvider = cosmosUniqueKeyPolicyProvider;
+        }
```

Comment:
```suggestion
            ICosmosUniqueKeyPolicyProvider cosmosUniqueKeyPolicyProvider)
        {
            _containerNameProvider = containerNameProvider;
            _cosmosPartitionKeyPathProvider = cosmosPartitionKeyPathProvider;
            _cosmosUniqueKeyPolicyProvider = cosmosUniqueKeyPolicyProvider;
        }
```

Inconsistent styling.